src/components/TableWithInlineEditing.js

```diff
@@ -77,14 +77,17 @@ function TableWithInlineEditing({ content, columnsInfo, updateContent, allowedTo
 
 TableWithInlineEditing.propTypes = {
     content: PropTypes.arrayOf(
-        PropTypes.shape({ id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]) })
+        PropTypes.shape({
+            id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+            title: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
```

Comment:
Is the title meant to be here, like provide a title and id for each row in the table? I'll have to merge with my changes at some point so if not I can remove it then

Reply:
I'm not totally sure. Feel free to rewrite when you next merge.